### PR TITLE
Unittests done for sprint 5

### PR DESCRIPTION
**Pull request by ti84plusprograms**

### CODE REVIEW

**Review by ti84plusprograms — COMMENTED** [review]
Good job completing it!!

File: app/src/androidTest/java/com/example/demo_2340/EnemyInstantiationTest.java
Comment (rnandakumar2001) [inline app/src/androidTest/java/com/example/demo_2340/EnemyInstantiationTest.java:1]: looks good

File: app/src/androidTest/java/com/example/demo_2340/PowerUpEffects.java
Comment (rnandakumar2001) [inline app/src/androidTest/java/com/example/demo_2340/PowerUpEffects.java:1]: nice job with the unit tests

**Review by rnandakumar2001 — APPROVED** [review]
nice job ary

**Pull request merged**